components/org.wso2.carbon.sp.jobmanager.core/src/main/java/org/wso2/carbon/sp/jobmanager/core/dbhandler/StatusDashboardManagerDBHandler.java

```diff
@@ -0,0 +1,199 @@
+    private String checkTableQuery;
+    private String deleteQuery;
+    private String insertQuery;
+    private HikariDataSource dataSource;
+    private Map<String, String> managerAttributeMap;
+    private QueryManager managerQueryManager;
+
+    public StatusDashboardManagerDBHandler() {
+        dataSource = ManagerDataHolder.getInstance().getDashboardManagerDataSource();
+        Connection connection = null;
+        if (dataSource != null) {
+            try {
+                connection = dataSource.getConnection();
+                DatabaseMetaData databaseMetaData = connection.getMetaData();
+                managerQueryManager = new QueryManager(databaseMetaData.getDatabaseProductName(), databaseMetaData
+                        .getDatabaseProductVersion());
+                managerAttributeMap = DBTableUtils.getInstance().loadManagerConfigTableTuples(managerQueryManager);
+                deleteQuery = managerQueryManager.getQuery(SQLConstants.DELETE_QUERY);
+                insertQuery = managerQueryManager.getQuery(SQLConstants.INSERT_QUERY);
+                checkTableQuery = managerQueryManager.getQuery(SQLConstants.ISTABLE_EXISTS_QUERY);
+                createTableQuery = managerQueryManager.getQuery(SQLConstants.CREATE_TABLE);
+            } catch (SQLException | QueryMappingNotAvailableException | ConfigurationException ex) {
+                throw new StatusDashboardRuntimeException("Error while initializing the connection ", ex);
+            } finally {
+                cleanupConnection(connection);
+            }
+            createTables();
+        } else {
+            throw new RDBMSTableException(DATASOURCE_ID + " Could not find.Hence cannot initialize the status "
+                                                  + "dashboard "
+                                                  + "Please check whether database is available");
+        }
+    }
+
+    /**
+     * Method which can be used to clear up and ephemeral SQL connectivity artifacts.
+     *
+     * @param conn {@link Connection} instance (can be null)
+     */
+
+    private void cleanupConnection(Connection conn) {
+        if (conn != null) {
+            try {
+                conn.close();
+                if (logger.isDebugEnabled()) {
+                    logger.debug("Closed Connection in Manager DB");
+                }
+            } catch (SQLException e) {
+                if (logger.isDebugEnabled()) {
+                    logger.debug("Error while closing the connection in manager DB");
+                }
+            }
+        }
+    }
+
+    /**
+     * Method which can be used to close the prepared statement.
+     */
+
+    private void closePreparedStatement(PreparedStatement statement) {
+        try {
+            if (statement != null) {
+                statement.close();
+            }
+        } catch (SQLException e) {
+            logger.debug("Error occurred while closing the statement " + e.getMessage(), e);
```

Comment:
always printing the log as a debug log?Instead of appending e.getMessage(), add useful info to your error log and the stack trace will be printed any way here.